### PR TITLE
First draft on a released based deployment workflow

### PR DESCRIPTION
**Pull request by jonasanker**

### CODE REVIEW

File: .github/workflows/deploy-dev.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Deploy to dev
```
Comment (duckth) [inline .github/workflows/deploy-dev.yml:1]: Minor, but the indentation seems off in this file (4 spaces) - other files seem to use 2 spaces so let's be consistent

File: .github/workflows/deploy-prd.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: Deploy to prd
```
Comment (duckth) [inline .github/workflows/deploy-prd.yml:1]: ditto

File: .github/workflows/pullrequest.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: Pull Request
```
Comment (duckth) [inline .github/workflows/pullrequest.yml:1]: also here

**Review by duckth — CHANGES_REQUESTED** [review]

**Comment by jonasanker** [comment]
@duckth I can make the identation changes next week but if you are bored tomorrow, feel free to perform the changes yourself and merge this PR :)

**Comment by duckth** [comment]
> @duckth I can make the identation changes next week but if you are bored tomorrow, feel free to perform the changes yourself and merge this PR :)

Sure thing, I'll do it 👍

**Review by duckth — APPROVED** [review]

**Pull request merged**